docs/design-time-builds.md

```diff
@@ -39,25 +40,25 @@ If you've determined that your target needs to run in a design-time build, using
 ```
 The `AddAdditionalReferences` target will run in both normal builds _and_ design-time builds, leading to consistent results between them.
 
-### Determining whether a target is run in a design-time build
+### Determining whether a target is running in a design-time build
 
-Use the `DesignTimeBuild` (CPS-based projects) and `BuildingProject` (legacy project system) properties to differentiate between when a target is run in a design-time build versus a normal build. This can be used to avoid expensive calculations or work that is only needed for a normal build.
+Use both the `DesignTimeBuild` (CPS-based projects) and `BuildingProject` (legacy project system) properties to determine whether a target is running in a design-time build or a normal build. This can be used to avoid expensive calculations or work that is only needed for a normal build, helping to keep the IDE responsive.
 
 ``` XML
   <Target Name="AddAdditionalReferences" BeforeTargets="ResolveAssemblyReferences">
-     <PropertyGroup Condition="$(DesignTimeBuild) == true OR $(BuildingProject) != true">
+     <PropertyGroup Condition="$(DesignTimeBuild) == 'true' OR $(BuildingProject) != 'true'">
```

Comment:
Either quotes are added on both sides, or neither.

Reply:
I didn't realise you could omit the quotes. I'll add them on both sides. My search through the code found no instances of `== true` or `== false` in project files.

Reply:
I think this was contributed by external contributor. You can omit the quotes but I can't remember the rules offhand. I'd probably always include the quotes.